wagtail/admin/templates/wagtailadmin/edit_handlers/help_panel.html

```diff
@@ -0,0 +1,6 @@
+<fieldset>
+	{% if self.heading %}
+    	<legend>{{ self.heading }}</legend>
+    {% endif %}
+    <div class="{{ self.classname }}">{{ self.content }}</div>
```

Comment:
Could you try to put HTML tags in `content` to see how it behaves? Given the default behaviour of the Django template language escaping HTML, I think there should be a `|safe` filter here if we want to accept raw HTML.